samples/meetings-notification/csharp/README.md

```diff
@@ -15,22 +15,23 @@ urlFragment: officedev-microsoft-teams-samples-meetings-notification-csharp
 
 # Meetings Notification
 
-This sample illustrates how to implement [Targeted In-Meeting Notification](https://learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/meeting-apps-apis?branch=pr-en-us-7615&tabs=dotnet#targeted-meeting-notification-api)  In-Meeting Experience.
+This sample illustrates how to implement [In-Meeting Notification](https://learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/meeting-apps-apis?branch=pr-en-us-7615&tabs=dotnet#send-an-in-meeting-notification) and [Targeted In-Meeting Notification](https://learn.microsoft.com/microsoftteams/platform/apps-in-teams-meetings/meeting-apps-apis?branch=pr-en-us-7615&tabs=dotnet#targeted-meeting-notification-api) for scheduled meetings.
 
 ## Included Features
 * Bots
 * In-Meeting Notifications
+* Targeted In-Meeting Notifications
 * Adaptive Cards
 * RSC Permissions
 
 ## Interaction with app
 
-![In-Meeting Notification](InMeetingNotifications/Images/MeetingNotification.gif)
+![Meetings Notification](InMeetingNotifications/Images/MeetingNotification.gif)
```

Comment:
What is the correct here In-Meeting or Meetings?

Reply:
Since this sample supports both in-meeting and targeted notifications so used meetings notification